tests/test_probes.py

```diff
@@ -5,6 +5,8 @@
 
 from chaosinstana.probes import get_all_events_in_window
 from chaosinstana.probes import convert_time
+from chaosinstana.probes import has_severity
```

Comment:
we tend to regroup all imports from a given module on one single "from ... import ..." statement